revolv/project/tests/tests_general.py

```diff
@@ -129,6 +130,48 @@ def test_get_drafted(self):
         self.assertEqual(context[0].org_name, "Fire Emblem")
 
 
+class CategoryTest(TestCase):
+    """Tests that category selection and updating work with projects."""
+
+    def test_update_category(self):
+        """ Tests that updating a single projects category works """
+        project = Project.factories.base.create(funding_goal=200.0, amount_donated=0.0, amount_left=200.0)
+        category1 = Category.factories.base.create()
+        category2 = Category.factories.base.create()
+        # tests that associating two categories with the project works
+        project.update_categories(Category.valid_categories[:2])
+        self.assertEqual(len(project.category_set.all()), 2)
+        self.assertItemsEqual(project.category_set.all(), [category1, category2])
+        # tests that having no categories with the project works
+        project.update_categories([])
+        self.assertEqual(len(project.category_set.all()), 0)
+
+    def test_update_category_multiple_projects(self):
+        """ Tests that removing categories from a project does not affect other projects """
```

Comment:
Random thing, use `Test` instead of `Tests` in the docstring for consistent style